src/core/execute.h

```diff
@@ -82,6 +82,7 @@ struct ExecCommand {
         ExecStatus exec_status;
         LIST_FIELDS(ExecCommand, command); /* useful for chaining commands */
         bool ignore;
+        bool privileged;
```

Comment:
not that it really matters, but I figure we can set add `:1` to both bools now and make them a bit field.